security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,32 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-3"]
+== OpenShift File Integrity Operator 1.3.3
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.3:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5652[RHBA-2023:5652 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+This update addresses a CVE in an underlying dependency.
+
+[id="file-integrity-operator-1-3-3-new-features-and-enhancements"]
+=== New features and enhancements
+
+* You can install and use the File Integrity Operator in an {product-title} cluster running in FIPS mode.
+
+[IMPORTANT]
+====
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see (link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode])
+====
+
+[id="file-integrity-operator-1-3-3-bug-fixes"]
+=== Bug fixes
+
+* Previously, FIO rprivate default mount propagation in combination with `hostPath: path: /` volume mounts of pods created would break the CSI driver relying on multipath.  (link:https://issues.redhat.com/browse/OCPBUGS-14947[*OCPBUGS-14947*])
```

Comment:
typo, `rprivate`. Also two spaces after `multipath.`

Reply:
Now, the FIO pods have the HostToContainer mount propagation, should never prevent the unmounting of a CSI volume.
https://access.redhat.com/solutions/7017081 this one may explain this issue better.
This sentence(`FIO rprivate default mount propagation in combination with `hostPath: path: /` volume mounts of pods created would break the CSI driver relying on multipath`) is not easy to read. Maybe better to update to "Previously, some FIO pods with default mount propagation in combination with `hostPath: path: /` volume mounts would break the CSI driver relying on multipath"?

Reply:
Thank you @sheriff-rh - I've resolved the typos and added "This problem has been fixed and the CSI driver works correctly."

Reply:
THANK YOU @xiaojiey ! That's very helpful!

Reply:
@xiaojiey I have adopted your suggested sentence and yes it does read much better!